### ISSUE TITLE
Remove deprecated warning in PHP 8.4.1

### ISSUE THREAD

**Issue by jonagoldman** [tpetry/laravel-postgresql-enhanced#102]
When using PHP 8.4.1, the next warnings are shown:

```
PHP Deprecated:  Tpetry\PostgresqlEnhanced\Schema\Builder::createBlueprint(): Implicitly marking parameter $callback as nullable is deprecated, the explicit nullable type must be used instead in /var/www/html/vendor/tpetry/laravel-postgresql-enhanced/src/Schema/Builder.php on line 34
PHP Deprecated:  Tpetry\PostgresqlEnhanced\Schema\BuilderDomain::createDomain(): Implicitly marking parameter $check as nullable is deprecated, the explicit nullable type must be used instead in /var/www/html/vendor/tpetry/laravel-postgresql-enhanced/src/Schema/BuilderDomain.php on line 34
```

**Comment by tpetry** [comment]
I update my test matrix today and fix the bug. Thanks for reporting. I hadn't time to upgrade to PHP 8.4 for my projects yet.

**Comment by tpetry** [comment]
The fix is now live with release `2.0.2`.
